### ISSUE TITLE
Change thumbnail image from SF program

### ISSUE THREAD

**Issue by cara-lgrow** [citizenlabsgr/adopt-a-drain#224]
The thumbnail image that shows up for the LGROW site is SF's image. Can we also change the text that goes along with it?

See James' link to the program in slack...SF's logo pops up, with canned text that is incorrect. 
We need this fixed before we can share the program anywhere online.

**Comment by Wilfongjt** [comment]
duplicate of 196
